bip-0003.md

```diff
@@ -105,7 +105,7 @@ following list and address each as appropriate.
   implementers and users should deal with these incompatibilities.
 * Reference Implementation — Where applicable, a reference implementation, test vectors, and documentation must be
   finished before the BIP can be given the status "Complete". Test vectors must be provided in the BIP or
-  as auxiliary files (see [Auxiliary Files](#auxiliary-files)) under an acceptable license. The reference implementation can be provided in the BIP, as an auxiliary file, or per reference to a pull request that is expected to remain available permanently.
+  as auxiliary files (see [Auxiliary Files](#auxiliary-files)) under an acceptable license. The reference implementation can be provided in the BIP as an auxiliary file, reference to a pull request (expected to remain available permanently), a new code repository, or similar.
```

Comment:
A specific branch seems more obvious to me than a new repository, but sure. This should be phrased in a way that makes clear that it’s expected to remain available permanently either way.